Box.V2/Managers/BoxUsersManager.cs

```diff
@@ -220,10 +220,46 @@ public async Task<bool> DeleteEmailAliasAsync(string userId, string emailAliasId
         /// Retrieves information about a user in the enterprise. Requires enterprise administration authorization.
         /// </summary>
         /// <param name="userId">The user identifier.</param>
+        /// <param name="fields">Attribute(s) to include in the response.</param>
         /// <returns>Returns the complete user object.</returns>
-        public async Task<BoxUser> GetUserInformationAsync(string userId)
+        public async Task<BoxUser> GetUserInformationAsync(string userId, IEnumerable<string> fields = null)
         {
-            BoxRequest request = new BoxRequest(_config.UserEndpointUri, userId);
+            //This is supposed to retrieve the "complete" user object according to the API, so it should probably return all of the actual fields available by default.
```

Comment:
This isn't really the way the SDK is intended to work — it's not guaranteed to produce all possible fields by default.  If additional fields beyond the ones returned from the API by default are required, they should be passed in explicitly via `fields`.  Please remove the section constructing a list of all possible fields below.

Reply:
Then I'd suggest simultaneously updating the description of  GetUserInformationAsync as it currently says it "returns the complete user object": https://github.com/box/box-windows-sdk-v2/blob/ca3034ee7663d95c20ce51362d6610f298197d65/Box.V2/Managers/BoxUsersManager.cs#L223

Didn't want to deviate from the existing documentation, hence that inclusion.

Should we call it the "default user object" instead?  Because it's not the mini-user object, but it's also not complete.  I'll be suggesting an edit to the readme.io to make the distinction clearer between different object representations.